### ISSUE TITLE
Job for system-cloudinit@-var-tmp-hostname.yml.service failed

### ISSUE THREAD

**Issue by canadatom** [rancher/rancher#1281]
Running with vagrant failed with the following message
```
==> rancher: Machine booted and ready!
==> rancher: Setting hostname...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

systemctl start system-cloudinit@-var-tmp-hostname.yml.service

Stdout from the command:



Stderr from the command:

Job for system-cloudinit@-var-tmp-hostname.yml.service failed because a configured resource limit was exceeded. See "systemctl status system-cloudinit@-var-tmp-hostname.yml.service" and "journalctl -xe" for details.
```

**Comment by deniseschannon** [comment]
The Vagrantfile needs to be updated. We'll be updating it with #1361 in the next week.

**Comment by imikushin** [comment]
should be fixed by #1615

**Comment by deniseschannon** [comment]
Fixed by #1615
